packages/webview_flutter/webview_flutter_wkwebview/test/v4/webkit_navigation_delegate_test.dart

```diff
@@ -148,6 +150,7 @@ void main() {
         callbackError.errorType,
         WebResourceErrorType.webContentProcessTerminated,
       );
+      expect(callbackError.isForMainFrame, true);
```

Comment:
All three cases are `true`; should there be any tests covering `false`?

Reply:
According to https://github.com/flutter/plugins/pull/3078, iOS only calls errors on the main frame. So, it should be true in each of these scenarios.